projectsinia/Assets/Assets/Scripts/PlayerMovement.cs

```diff
@@ -9,6 +9,7 @@ public class PlayerMovement : MonoBehaviour
 
 	public MovementInfoScriptableObject Data;
 	public PlayerAnimator animController;
+	public DashTrail dt;
```

Comment:
Ideally, you want to give variables self-explanatory names. If you write out this short name here for example, you will have an easier time knowing the variables purpose upon next code change